script.js

```diff
@@ -0,0 +1,108 @@
+
+const taskInput = document.getElementById("todoInput");
+const addTaskBtn = document.getElementById("addButton"); //왜 안됌>?
+const pendingTasksList = document.getElementById("todosPendingList");
+const completedTasksList = document.getElementById("todosFinishedList");
+
+
+//create New todo format
+let createNewTask = function (todo) {
+    let newTodo = document.createElement("li");
+    let label = document.createElement("label");
+    let deleteBtn = document.createElement("button");
```

Comment:
생각보다` let`을 사용해야만 하는 상황이 흔치 않습니다.` const`로 쓸 수 있는 것들은 `const`로 사용해주세요~(아래 코드들도 마찬가지)
const를 사용하면 의도치 않은 재할당을 방지해 주기 때문에 보다 안전하게 작성할 수 있습니다. const와 let의 차이점에 대해서 알아보시면 좋을 것 같아요~

```suggestion
const createNewTask = function (todo) {
    const newTodo = document.createElement("li");
    const label = document.createElement("label");
    const deleteBtn = document.createElement("button");
```

Reply:
저도 이번기회를 통해 변수 선언에 대해 잘 배울 수 있었습니다.
https://velog.io/@bathingape/JavaScript-var-let-const-%EC%B0%A8%EC%9D%B4%EC%A0%90
제가 참고한 정보입니다!